.github/workflows/build-library.yml

```diff
@@ -6,10 +6,20 @@ on:
         branches:
             - "dev"
             - "releases/**"
+        paths:
+            - "data-processing-lib/**"
+            - "!**.md"
+            - "!**/doc/**"
+            - "!**/.gitignore"
     pull_request:
         branches:
             - "dev"
             - "releases/**"
+        paths:
+            - "data-processing-lib/**"
```

Comment:
"kfp/kfp_support_lib/**" and "kfp/kfp_ray_components/**"

Reply:
Why? This workflow does not touch the kfp subdir